solana-programs/reward-manager/program/src/processor.rs

```diff
@@ -158,8 +181,43 @@ impl Processor {
         Ok(())
     }
 
+    fn process_delete_sender_public<'a>(
+        program_id: &Pubkey,
+        reward_manager_info: &AccountInfo<'a>,
+        sender_info: &AccountInfo<'a>,
+        refunder_info: &AccountInfo<'a>,
+        signers_info: Vec<&AccountInfo>,
+        instructions_info: &AccountInfo<'a>,
+    ) -> ProgramResult {
+        assert_owned_by(reward_manager_info, program_id)?;
+        assert_owned_by(sender_info, program_id)?;
+
+        let reward_manager = RewardManager::unpack(&reward_manager_info.data.borrow())?;
+        let sender_account = SenderAccount::unpack(&sender_info.data.borrow())?;
+
+        if signers_info.len() < reward_manager.min_votes.into() {
+            return Err(AudiusProgramError::NotEnoughSigners.into());
+        }
+
+        check_secp_instructions(
+            program_id,
+            reward_manager_info.key,
+            instructions_info,
+            signers_info.clone(),
+            signers_info.len(),
+            sender_account.eth_address,
+            DELETE_SENDER_MESSAGE_PREFIX,
+        )?;
+
+        assert_account_key(reward_manager_info, &sender_account.reward_manager)?;
+
+        Self::transfer_all(sender_info, refunder_info)?;
```

Comment:
if my understanding is correct, this should basically remove all balance from the existing sender_info account to be deleted and can longer be used?

Reply:
Yes, right
It will move all the SOL to refunder account and sender account will not be able to pay rent anymore and will not be rent exempt so it will be removed

Reply:
perfect, thanks for clarifying!